cves/kernel/CVE-2012-2375.yml

```diff
@@ -387,38 +424,38 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
```

Comment:
I feel like you could definitely sift through these and find them.  Is np a framework?  I also think input should be distrusted in the case of bad input breaking the system, similarly complex inputs and potentially security by obscurity can be pulled in.  Do some more research here.

Reply:
updated